jenkins_pipelines/scripts/json_generator/repository_versions/v43_nodes.py

```diff
@@ -101,13 +101,13 @@
 
 # Dictionary for SUMA 4.3 Server and Proxy
 v43_nodes: Dict[str, Dict[str, str]] = {
-    "server": {"/SUSE_Updates_SLE-Module-SUSE-Manager-Server_4.3_x86_64/",
-               "/SUSE_Updates_SLE-Product-SUSE-Manager-Server_4.3_x86_64/",
+    "server": {"/SUSE_Updates_SLE-Module-SUSE-Manager-Server_4.3-LTS_x86_64/",
+               "/SUSE_Updates_SLE-Product-SUSE-Manager-Server_4.3-LTS_x86_64/",
```

Comment:
I'm not sure if we need to keep both or not.
But if you have the details about it, then LGTM.